gradle/libs.versions.toml

```diff
@@ -56,6 +56,8 @@ awssdk-core = { module = 'software.amazon.awssdk:sdk-core' }
 
 micronaut-aws-bom = { module = 'io.micronaut.aws:micronaut-aws-bom' }
 
+javax-annotation-api = { module = "javax.annotation:javax.annotation-api" }
```

Comment:
this shouldn't be used

Reply:
Any idea how to get around grpc annotating classes it creates with `@javax.annotation.Generated`?

Reply:
Seems no way until this issue will be resolved: https://github.com/grpc/grpc-java/issues/9179